### DISCUSSION TITLE
Document context for terms that designate, and other questions

### DISCUSSION THREAD

**Discussion by alanruttenberg** [CommonCoreOntology/CommonCoreOntologies#402] [Announcements]
>designates:
>x designates y iff x is an instance of an Information Content Entity, and y is an instance of an Entity, such that given some context, x uniquely distinguishes y from other entities.

Without giving more information about context,  the definition is trivially satisfied if the context includes only a single entity. In all cases it seems necessary to say something about the intended context.

- A person's name does not uniquely distinguish a person unless the context is very small (some, but not all groupings of people)
- Artifact Model Name:  A model does not uniquely distinguish something. There were many of the model of my TV made. However, on reflection I realize I don't know what sense of model you mean. Do you mean what might be called a model number e.g.  Koss **KEB15i** In-Ear Headphone, or do you mean a model as in model aircraft. Model names and numbers can be duplicated across different types of products. Is the context Artifacts? 
- Designative name: An example of usage and of non-usage would be helpful. 
- Spatial Region Identifier: Probably should be site identifier.
- The Temporal * Identifer: Are not unique unless it is specified that AM/PM is included as appropriate as well as time zone. The hour, minute, second terms are not unique unless a date and time zone is included. Unless, I suppose, the context for hour was the date, context for the minute the hour, context for second the minute?
- Abbreviated Name: not unique. Not unusual to have two "Joe"s in a room. What is the context?
- Why aren't all the siblings of Designative Names, Designative Names. I could imagine QR codes not being designative names but the siblings don't seem to be so.

**Comment by mark-jensen** [comment]
I'm not clear about what more needs to be said at the level of the term in the ontology, other than what is currently mentioned about designation happening within a context. Can you be more specific? A few initial points below.

> In all cases it seems necessary to say something about the intended context.

Agrreed. And typically, the context is captured as part of the data ingest and transformation, eg., this csv ingested on this date, data about satellite feeds per day and location, vehicles assigned to a particular military unit during a temporal interval, etc. The context gets embedded in the full graph that spins out from that transformation. For example, the context associated with all the serial numbers as designators of vehicles will be embedded in a graph that links those serial numbers to the vehicles and who has possession of them, their operational status, administrating unit, and so on. If just the few triples associated with the vehicle and its designator are pulled from the graph, then yes, context gets lost. But, that is always a problem, isn't it? 


> A person's name does not uniquely distinguish a person unless the context is very small (some, but not all groupings of people)

Yes, true. But see above. Within a data record, even one that contains multiple "Joe's", there will (or should) be other contextual facts represented in the graph. And if not, it seems to me a problem with the data or the person modeling the data.

> A model does not uniquely distinguish something. 

Correct, an Artifact Model does not uniquely distinguish. It _prescribes_ the materials, qualities, and dispositions of an entity. But, artifact models (as directive ICEs) _are_ uniquely named, like novels or plan specifications, something is used to identify them. I believe the model number can desigante the artifact model sometimes, but not always.

>Why aren't all the siblings of Designative Names, Designative Names. I could imagine QR codes not being designative names but the siblings don't seem to be so.

There may be some cleaning up to do here. But, the 4 siblings of DN are all equivalencies, and strictly not necessary, they are defined by what kind of thing they designate, e.g. Day or Year. I'd be ok with deleting them, save the couple subclasses that don't have axioms.
